scripts/analysis.py

```diff
@@ -1091,32 +1034,229 @@ def plot_power(cursor):
+                                               'qualid') +
+                                   ' GROUP BY time, qualid').fetchall()
+
+    compositions = cursor.execute('SELECT qualid, nucid, massfrac '
+                                  'FROM compositions').fetchall()
+
+    init_year, init_month, duration, timestep = get_sim_time_duration(cursor)
+    transactions = get_isotope_transactions(resources, compositions)
+    waste_dict = get_waste_dict(transactions.keys(),
+                                transactions.values()[0],
+                                transactions.values()[1],
+                                duration)
+
+    if influx_bool is False:
+        stacked_bar_chart(waste_dict, timestep,
+                          'Years', 'Mass [kg]',
+                          title, outputname, init_year)
+    else:
+        multi_line_plot(waste_dict, timestep,
+                        'Years', 'Mass [kg]',
+                        title, outputname, init_year)
+
+
 if __name__ == "__main__":
     file = sys.argv[1]
     con = lite.connect(file)
     with con:
         cur = con.cursor()
         init_year, init_month, duration, timestep = get_sim_time_duration(cur)
-        plot_power(cur)
-        waste_dict = total_waste_timeseries(cur)
-        multi_line_plot(waste_dict, timestep,
+        mix_ratio(cur, 'cfr_fresh',
+                  'cfr_spent', 'depleted_u', [94, 95, 96, 97])
+        conv_ratio(cur, 'cfr_fresh', 'cfr_spent', True)
+        # get fuel source and inventory
+        fuel_dict = where_comm(
+            cur, 'mox', ['mox_uox_fuel_fab', 'mox_mixer'])
+        fuel_dict['from_spent_uox'] = fuel_dict.pop('mox_uox_fuel_fab')
+        fuel_dict['from_spent_mox'] = fuel_dict.pop('mox_mixer')
+        stacked_bar_chart(fuel_dict, timestep,
+                          'Years', 'Mass[MTHM]',
+                          'Total Fuel Mass vs Time',
+                          'where_fuel',
+                          init_year)
+
+        # get pu demand
+        demand = collections.OrderedDict()
+        demand['pu_from_legacy'] = [
+            i * .09 for i in fuel_dict['from_spent_uox']]
+        demand['pu_from_spent_mox'] = [
+            i * .09 for i in fuel_dict['from_spent_mox']]
+        total_mox = (
+            [x + y for x, y in zip(fuel_dict['from_spent_uox'], fuel_dict['from_spent_mox'])])
+        demand['pu_total'] = [i * .09 for i in total_mox]
+        demand['tailings'] = [i * .91 for i in total_mox]
+        multi_line_plot(demand, timestep,
                         'Years', 'Mass[MTHM]',
-                        'Total Waste Mass vs Time',
-                        'total_Waste',
+                        'Total Demand vs Time',
+                        'demand',
                         init_year)
 
-        fuel_dict = fuel_usage_timeseries(cur, ['uox', 'mox','fr_fuel'])
+        # get power / number timeseries
+        plot_power(cur)
 
+        # get reprocessing waste
```

Comment:
I agree but this function is there like a wrapper function, that just gets and plots the capacity and number of reactors over time.
Since we've needed to do that repeatedly, I added it to save time.